build/processes/build-cleanup.js

```diff
@@ -0,0 +1,64 @@
+/*!***************************************************
+ * diacritics
+ * http://diacritics.io/
+ * Copyright (c) 2016–2018 The Diacritics Authors
+ * Released under the MIT license https://git.io/vXg2H
+ *****************************************************/
+'use strict';
+const fs = require('fs'),
+  und = require('../../src/und/und.json'),
+  Utils = require('./utils');
+
+class Cleanup {
+  /**
+   * Constructor
+   */
+  constructor() {
+    this.buildFile = './build/build-undetermined.js';
+    this.run();
+  }
+
+  /**
+   * Remove existing diacritics from undetermined list
+   * @param {object} output - The database file content
+   * @param {object} charList - A Set object containing all diacritics from the
+   * database
+   * @return {object}
+   */
+  removeUndeterminedDuplicates(output, charList) {
+    [...charList].forEach(char => {
+      if (output.und.und.data[char]) {
+        delete output.und.und.data[char];
+        delete und.data[char];
+        this.buildContent = this.buildContent.replace(char, '');
```

Comment:
`this.buildContent` seems not to be declared in the constructor. Please always declare variables in the constructor, otherwise you don't know what's stored inthe class without digging into each method.